README.md

```diff
@@ -1462,6 +1462,48 @@ jobs:
           publish-summary: false
 ```
 
+### Automatic PR number & URL detection
+
+The PR number and URL can be automatically detected if you pass `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` via the workflow `env`. When set,
+this value enables the Action to perform additional logic that grabs the related PR number and URL (if they exist) and sets them in the
+environment variables `CYPRESS_PULL_REQUEST_ID` and `CYPRESS_PULL_REQUEST_URL`, respectively.
+
+Example workflow using the new variables:
```

Comment:
```suggestion
Example workflow using the variables:
```

Remove "new" because the README always refers only to the current version unless otherwise stated.